.travis.yml

```diff
@@ -18,6 +18,7 @@ install:
   - pip install flake8
```

Comment:
IMO isort shouldn't be added on `requirements-test.txt` file because it's not used in the project's test suite. Instead it should be installed here with other checkers such as `flake8`.

Reply:
Done